src/main/resources/templates/resource2.html

```diff
@@ -0,0 +1,250 @@
+												<button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+												<button type="button" class="btn btn-standard">Save</button>
+											</p>
+										</div>
+										<div class="modal-footer">
+											<button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+										</div>
+									</div>
+								</div>
+							</div>
+						</div>
+						<td>vsftpd</td>
+						<td>Backdoor Trojan</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+
+						</td>
+					</tr>
+					<tr>
+						<td>2</td>
+						<td>CVE-2014-6271</td>
+						<td>Bash Code Injection</td>
+						<td>ShellShock</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+						<td>3</td>
+						<td>CVE-2014-0160</td>
+						<td>OpenSSL 'Heartbleed'</td>
+						<td>Heartbleed</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+						<td>4</td>
+						<td>CVE-2003-0352</td>
+						<td>Buffer Overrun in RPC</td>
+						<td>Microsoft RPC DCOM Interface Overflow</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+						<td>5</td>
+						<td>OSVDB-16373</td>
+						<td>gIFTPD dupescan buffer overflow</td>
+						<td>SSH CRC-32 Compensation Attack Detector</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+						<td>6</td>
+						<td>OSVDB-60979</td>
+						<td>HTTPd Remote Heap Overflow</td>
+						<td>Null HTTPd Remote Heap Overflow Vulnerability</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+						<td>7</td>
+						<td>EDB-816</td>
+						<td>Anything to PostScript</td>
+						<td>GNU a2ps Local Exploit </td>
+						<td><button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+
+						</td>
+					</tr>
+
+						</table>
+				</div>
+				<!--<p>Some content.</p>-->
+			</div>
+			<div id="bigdata" class="tab-pane fade">
+				<!--<h3>Menu 1</h3>-->
+				<!--<p>Some content in menu 1.</p>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)